packages/sdk/src/entities/BorrowingVault.ts

```diff
@@ -275,6 +298,30 @@ export class BorrowingVault {
     return { deposit, borrow };
   }
 
+  /**
+   * Returns a stream of deposit and borrow balances for an account.
+   *
+   * @param account - user address, wrapped in {@link Address}
+   * @throws if {@link setConnection} was not called beforehand
+   */
+  getBalancesStream(account: Address): Observable<AccountBalances> {
+    invariant(this.contract && this.wssProvider, 'Connection not set!');
+    const filters = [
```

Comment:
We don't need `flashclose | liquidate` in our filters ?

Reply:
good catch about liquidate